### ISSUE TITLE
Bug: Nimdoc comments for objects are lost in serializable macro

### ISSUE THREAD

**Issue by zintrepid** [xomachine/NESM#19]
The comments I leave for objects are not present in the generated documentation.

Example:
```
import NESM
serializable:
  type MyType = object
    ## Documentation on MyType
    field1: string  ## Documentation on field1
```

When I run `nim doc` on this, I expect to see both nimdoc comments, but only the field comment is present.

**Comment by zintrepid** [comment]
I propose #20 to address this.

**Comment by zintrepid** [comment]
This was fixed with 94aed1a.
